incubator/binding-risingwave/src/main/java/io/aklivity/zilla/runtime/binding/risingwave/internal/stream/RisingwaveProxyFactory.java

```diff
@@ -887,13 +902,13 @@ private void onApplicationFlush(
             switch (pgsqlFlushEx.kind())
             {
             case PgsqlFlushExFW.KIND_TYPE:
-                server.doApplicationFlush(traceId, authorization, extension);
+                onCommandType(traceId, authorization, pgsqlFlushEx);
```

Comment:
Does `onCommandXxx` describe what's happening here, or are these something else?
Perhaps `onAppTypeFlush`, `onAppCompletionFlush`, `onAppReadyFlush`?

Btw, the modern binding codebases use `onAppXxx` and `onNetXxx` instead of `onApplicationXxx` and `onNetworkXxx` so let's go with the more modern naming convention here as well.

Reply:
Thanks I was really struggling naming them.